test/samplers/test_samplers.py

```diff
@@ -321,14 +343,19 @@ def test_sampling_range_default_regular_sampler(sampler):
         partial(
             clips_at_regular_indices, sampling_range_start=-1, sampling_range_end=1000
         ),
-        # Note: the hard-coded value of sampling_range_start=12 is because we know
+        # Note: the hard-coded value of sampling_range_start=13 is because we know
```

Comment:
Assuming *exactly* 13.01 corresponds to `end_stream_seconds`, then `13.01` would be an invalid value here.

We just need to set a value that is < `end_stream_seconds` so that the clip start is valid, but large enough so that the clip span is such that it goes beyond `end_stream_seconds` and thus triggers the "error" policy.

Similarly for the index-based case, we set this value to `-1`, thus enforcing the clip to start on the last frame.

I will slightly update the comment so as to not claim that the video duration is exactly 13.01.